vizro-core/src/vizro/models/_navigation/_accordion.py

```diff
@@ -26,17 +26,17 @@ class Accordion(VizroBaseModel):
     _validate_pages = validator("pages", allow_reuse=True, always=True)(_validate_pages)
 
     @_log_call
-    def build(self):
+    def build(self, page_id):
```

Comment:
@antonymilne - we usually don't have any arguments for the build method. I don't know if we should deviate from that. Here, it is required to know what the currently active page is such that a different CSS class is provided in the accordion.

Is this better stored in a PrivateAttr or a dcc.store, as knowing the currently selected page seems helpful in several places?